bin/cdklocal

```diff
@@ -451,6 +451,13 @@ const patchPre_2_14 = () => {
   applyPatches(provider, CdkToolkit, SDK, ToolkitInfo);
 };
 
+const configureEnvironment = () => {
+  // This _must_ use localhost.localstack.cloud as we require valid subdomains of these paths to
+  // resolve. Unfortunately though `curl` seems to support subdomains of localhost, the CDK does not.
+  process.env.AWS_ENDPOINT_URL_S3 = process.env.AWS_ENDPOINT_URL_S3 || `${PROTOCOL}://s3.localhost.localstack.cloud:${EDGE_PORT}`;
```

Comment:
I forgot to share, but by default, we are also using `s3.localhost.localstack.cloud` for `tflocal`